bin/git-rename-remote

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+old=$1
+new=$2
+
+test -z $old && echo "old remote name required." 1>&2 && exit 1
+test -z $new && echo "new remote name required." 1>&2 && exit 1
+
+if git config --get "remote.$new.fetch"; then
```

Comment:
To avoid the confusing and useless output like `+refs/heads/*:refs/remotes/origin/*`, we should redirect the stdout to `/dev/null`.